core/controllers/profile.py

```diff
@@ -382,10 +382,10 @@ def get(self):
             temp_file, mode='w', compression=zipfile.ZIP_DEFLATED) as zfile:
             zfile.writestr('oppia_takeout_data.json', json.dumps(user_data))
             for image in user_images:
-                b64_png_no_header = image.b64_image_data.split(',')[1]
-                b64_png_no_header = python_utils.url_unquote_plus(
-                    b64_png_no_header)
-                b64_png_no_header = re.sub(r'\s', b'+', b64_png_no_header)
+                b64_png_no_header = re.sub(
+                    r'\s', b'+',
+                    python_utils.url_unquote_plus(
+                        image.b64_image_data.split(',')[1]))
```

Comment:
Should this be moved to utils.py and call it something like `get_base64_without_header`?

Reply:
Btw, what does "no header" mean exactly? Some explanation (in a comment) would be helpful.

(I can't tell whether the header is a oppia-specific concept or a general thing. If the latter, then utils.py makes sense; if the former, probably not.)

Reply:
By default the base64 image comes back in the format ```image/png, [base64data]```, so I'm removing the `image/png` header from the base65 string. I think it's more of a specific thing.

Reply:
Ah ok, definitely you want to add some explanatory comments here. Maybe also add a check that the part before the comma is what you expect it to be, otherwise log/throw an error.

Also I don't think this is generic enough to live in utils.py, it's quite a specific operation given that context. @vojtechjelinek would you agree?

Reply:
I mean the format that it comes back in is commonly used in frontend and HTML where base64 is prefixed with the filetype so that the browser compiler knows how to interpret it. We even do a similar kind of thing already in https://github.com/oppia/oppia/blob/03f7d149409ad1b67a31b616a2be1bca26e394ed/utils.py#L285-L290

Reply:
@vojtechjelinek That's a really good find, thanks! Let's try and reuse that function then.

Reply:
I updated the function and reused it.